.github/workflows/main.yml

```diff
@@ -110,11 +109,96 @@ jobs:
         name: android-build
         path: pronto_mia/build/app/outputs/apk/release/app-release.apk
         retention-days: 1
+
+  build-web-docker-master:
+    name: Push Docker master image to GitHub Container Registry
+    runs-on: ubuntu-latest
+    needs: test
+    if: github.ref == 'refs/heads/master'
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
       
-  documentation:
+      - name: Set up QEMU
```

Comment:
```suggestion
      - name: Setup QEMU
```

Reply:
All the other setup steps are named this way. Either change the others to "set up" or the new ones to "setup".